source/CCustomOpcodeSystem.cpp

```diff
@@ -2304,11 +2304,15 @@ namespace CLEO
 				return CCustomOpcodeSystem::ErrorSuspendScript(thread);
 			}
 
-			if(returnParamCount - 1 != declaredParamCount) // minus 'num args' itself

```

Comment:
Should we disable this check only in CLEO4 compat mode (.cs4 extension)?

Reply:
I don't know, perhaps we should hsve a separate doc file which would list all differences for .cs4